flaml/data.py

```diff
@@ -23,9 +23,11 @@
     SEQCLASSIFICATION,
     MULTICHOICECLASSIFICATION,
     TOKENCLASSIFICATION,
+    "mm_multi",
+    "mm_binary",
 )
 SEQREGRESSION = "seq-regression"
-REGRESSION = ("regression", SEQREGRESSION)
+REGRESSION = ("regression", "mm_regression", SEQREGRESSION)
```

Comment:
Rename "mm_regression" -> "multimodal-regression", define a static variable for it

Reply:
I'm afraid we cannot. By initializing the TextPredictor, this path is either defined by a user or created by AG to save the model.
Ref: [https://github.com/awslabs/autogluon/blob/0.4.0/text/src/autogluon/text/text_prediction/predictor.py#L56](url)